R/learner_flexsurv_surv_flexible.R

```diff
@@ -67,7 +67,7 @@ LearnerSurvFlexible = R6Class("LearnerSurvFlexible",
         id = "surv.flexible",
         packages = c("mlr3extralearners", "flexsurv", "pracma"),
         feature_types = c("logical", "integer", "factor", "numeric"),
-        predict_types = c("distr", "crank", "lp"),
+        predict_types = c("crank", "lp", "distr"),
```

Comment:
also update NEWS.md when you change things like that

Reply:
okay! This was part of this https://github.com/mlr-org/mlr3extralearners/issues/331, can do in parts since we changing some learners now